### PR TITLE
DEBIAN: Removed deb package dependencies

### PR DESCRIPTION
**Pull request by artemry-mlnx**
Signed-off-by: Artem Ryabov <artemry@mellanox.com>

## What
Changes:
* Removed deb package dependencies in scope of https://github.com/openucx/ucx/issues/5596

## Why ?
https://github.com/openucx/ucx/issues/5596

### CODE REVIEW

File: debian/control.in [hunk 1]
```diff
@@ -14,7 +14,7 @@ Homepage: http://www.openucx.org
 
 Package: @PACKAGE@
 Section: libs
-Depends: ${shlibs:Depends}, ${misc:Depends}
+#Depends: ${shlibs:Depends}, ${misc:Depends}
```
Comment (yosefe) [inline debian/control.in:17]: why comment out and not remove?

File: buildlib/docker-compose.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ services:
     image: centos7-mofed5.0-cuda10.1
     build:
       context: .
+      network: host
```
Comment (yosefe) [inline buildlib/docker-compose.yml:8]: why needed in this PR?

File: .gitignore [hunk 3]
```diff
@@ -87,3 +87,4 @@ GTAGS
 /modules
 *.swp
 compile_commands.json
+.idea/
```
Comment (yosefe) [inline .gitignore:90]: unrelated?

File: buildlib/docker-compose.yml (hunk 2 above)
Comment (artemry-mlnx) [reply buildlib/docker-compose.yml:8]: It's a part of the related verification.

File: debian/control.in (hunk 1 above)
Comment (artemry-mlnx) [reply debian/control.in:17]: If for some reasons we'll decide to revert it back for some reasons in the future.

File: .gitignore (hunk 3 above)
Comment (artemry-mlnx) [reply .gitignore:90]: Related to fix preparation.
It's a very minor change but if you insist it can be handled as a separate PR.

Comment (yosefe) [reply .gitignore:90]: should be another PR (update .gitignore)

File: buildlib/docker-compose.yml (hunk 2 above)
Comment (yosefe) [reply buildlib/docker-compose.yml:8]: what is it doing ? would AZP fail without this line, now that we removed deps?

File: debian/control.in (hunk 1 above)
Comment (yosefe) [reply debian/control.in:17]: we have git history for this.. we can add comment that the package does not have dependencies to allow running on different systems by relying on UCX dynamic modules

Comment (artemry-mlnx) [reply debian/control.in]: Removed.

File: buildlib/docker-compose.yml (hunk 2 above)
Comment (artemry-mlnx) [reply buildlib/docker-compose.yml]: It isn't used in AZP, just to build UCX build/test docker environment manually with docker-compose as described here https://github.com/openucx/ucx/blob/master/buildlib/azure-pipelines.md#release-images
I'll put it in a separate PR.

File: .gitignore (hunk 3 above)
Comment (artemry-mlnx) [reply .gitignore]: I'll put this into separate PR.

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
@artemry-mlnx can you pls change commit title to
"DEBIAN: Remove deb package dependencies"

**Comment by artemry-mlnx** [comment]
> @artemry-mlnx can you pls change commit title to
> "DEBIAN: Remove deb package dependencies"

Done.

**Comment by yosefe** [comment]
bot:pipe:retest

**Review by yosefe — APPROVED** [review]
